.github/ISSUE_TEMPLATE/REQUEST.yml

```
@@ -0,0 +1,41 @@
+name: "New Content Idea/Request 💡"
+description: Share an idea for new content.
+labels: [
+  "add new content"
+]
+body:
+  - type: markdown
+    attributes:
+      value: "# Reminder: If reporting a minor correction (e.g. typo), feel free to submit a PR directly!"
```

```suggestion
      value: "## Reminder: If reporting a minor correction (e.g. typo), feel free to submit a PR directly!"
```
Made it smaller, since technically this should be for new requests anyway.) Probably fine huge on the other one.